pumps/hybrid.go

```diff
@@ -128,8 +130,25 @@ func (p *HybridPump) Init(config interface{}) error {
 			p.trackAllPaths = trackAllPaths.(bool)
 		}
 
-		if storeAnalyticPerMinute, ok := meta["store_analytics_per_minute"]; ok {
-			p.storeAnalyticPerMinute = storeAnalyticPerMinute.(bool)
+		if storeAnalyticsPerMinute, ok := meta["store_analytics_per_minute"].(bool); ok {
+			p.storeAnalyticsPerMinute = storeAnalyticsPerMinute
+			p.aggregationTime = 1
+		} else {
+			aggregationTime, ok := meta["aggregation_time"].(int)
+			if !ok {
+				p.log.Warn("aggregation_time is not configured. The default value will be set to 60")
+				p.aggregationTime = 60
+			} else {
+				if aggregationTime > 60 || aggregationTime < 1 {
+					p.log.Warn("aggregation_time range is from 1 to 60. The default value will be set to 60")
+					p.aggregationTime = 60
+				}
+				p.aggregationTime = aggregationTime
+			}
+		}
```

Comment:
I think we can replace this with:

```
aggregationTime, ok := meta["aggregation_time"].(int)

if aggregationTime > 60 || aggregationTime < 1 {
					p.log.Warnf("aggregation_time should be between 1 and 60, Found: %v. The default value will be used  (60 minutes)",aggregationTime)
					p.aggregationTime = 60
				}

```

Reply:
Nice! It looks cleaner. But is it right not to check the value of "ok"?

Reply:
if ok is false, then `aggregationTime` will be zero, which is less than 1, hence it will enter into the conditional right?